From: Rashika Kheria <rashika.kheria@gmail.com>
Date: Sat, 14 Dec 2013 19:32:09 +0530
Subject: sb_edac: Mark get_mci_for_node_id as static
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
Git-commit: 8112c0cdf7b86f5917cdffcff4ffc477eaa62ed4
Patch-mainline: v3.14-rc1
References: fate#318187

This patch marks the function get_mci_for_node_id() as static because it
is not used outside of sb_edac.c.

Thus, it also eliminates the following warning:
drivers/edac/sb_edac.c:918:22: warning: no previous prototype for ‘get_mci_for_node_id’ [-Wmissing-prototypes]

Signed-off-by: Rashika Kheria <rashika.kheria@gmail.com>
Reviewed-by: Josh Triplett <josh@joshtriplett.org>
Link: http://lkml.kernel.org/r/0441f508186fc4eeabc8e9c3e4dde013d99405d4.1387029387.git.rashika.kheria@gmail.com
Signed-off-by: Borislav Petkov <bp@suse.de>
---
 drivers/edac/sb_edac.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: current/drivers/edac/sb_edac.c
===================================================================
--- current.orig/drivers/edac/sb_edac.c	2015-04-20 21:12:55.128122457 +0200
+++ current/drivers/edac/sb_edac.c	2015-04-20 21:13:25.292122403 +0200
@@ -915,7 +915,7 @@ static void get_memory_layout(const stru
 	}
 }
 
-struct mem_ctl_info *get_mci_for_node_id(u8 node_id)
+static struct mem_ctl_info *get_mci_for_node_id(u8 node_id)
 {
 	struct sbridge_dev *sbridge_dev;
 
